tests/test_fonts.py

```diff
@@ -0,0 +1,239 @@
+import pytest
```

Comment:
For import blocks like this I recommend following the best practice of grouping it into built-in, third-party and project dependencis, in this case: (io, textwrap), (lxml), (svglib). Maybe that's even in PEP8, but there are tools to autoformat code like this, like black and isort. I hope the code follows this in other places, already.